core-bundle/src/Controller/FrontendController.php

```diff
@@ -101,4 +101,26 @@ public function logoutAction(): void
     {
         throw new LogoutException('The user was not logged out correctly.');
     }
+
+    /**
+     * Generates a 1px transparent png image uncacheable response.
+     * This route can be used to include in e.g. a hidden <img> tag to force
+     * a request to the application. That way, cookies can be set even if
+     * the output is cached (used in core for the RememberMe cookie if
+     * the checkbox alwaysLoadFromCache is set).
+     *
+     * @Route("/_contao/check_cookies", name="contao_frontend_check_cookies")
+     */
+    public function checkCookiesAction(): Response
```

Comment:
...Action naming is deprecated afaik. Imho we should use an `__invoke`able controller for each action.